common_benchmark_suite/openxla/benchmark/comparative_suite/tf/scripts/generate_model_artifacts.py

```diff
@@ -125,27 +137,22 @@ def _parse_arguments() -> argparse.Namespace:
                       type=str,
                       default=".*",
                       help="The regex pattern to filter model names.")
-  parser.add_argument("--iree_import_tf_path",
-                      type=pathlib.Path,
-                      default="iree-import-tf",
-                      help="Path to `iree-import-tf`. Used to binarize mlir.")
-  parser.add_argument("--iree_opt_path",
-                      type=pathlib.Path,
-                      default=None,
-                      help="Path to `iree-opt`. Used to binarize mlir.")
   parser.add_argument(
       "--auto-upload",
       "--auto_upload",
       action="store_true",
       help=
       f"If set, uploads artifacts automatically to {GCS_UPLOAD_DIR} and removes them locally once uploaded."
   )
+  parser.add_argument("--generate_tflite",
+                      "--generate-tflite",
+                      action=argparse.BooleanOptionalAction,
```

Comment:
Maybe keep the consistency to use `action="store_true"`?

Reply:
Removed from this PR so that we can design this properly.